src/lib/utils.js

```diff
@@ -1,10 +1,11 @@
-export function post(endpoint, data) {
-	return fetch(endpoint, {
-		method: 'POST',
-		credentials: 'include',
-		body: JSON.stringify(data || {}),
-		headers: {
-			'Content-Type': 'application/json'
-		}
-	}).then((r) => r.json());
+export async function post(endpoint, data) {
+	const response = await fetch(endpoint, {
+        method: 'POST',
+        credentials: 'include',
+        body: JSON.stringify(data || {}),
+        headers: {
+            'Content-Type': 'application/json'
+        }
+    });
+    return await response.json();
```

Comment:
you need to just use tabs and not spaces
```suggestion
	const response = await fetch(endpoint, {
		method: 'POST',
		credentials: 'include',
		body: JSON.stringify(data || {}),
		headers: {
			'Content-Type': 'application/json'
		}
	});
	return await response.json();
```